lib/rex/proto/ms_dtyp.rb

```diff
@@ -30,62 +30,61 @@ class MsDtypAccessMask < BinData::Record
     bit2   :reserved1
     bit1   :ma
     bit1   :as
-    def bit_names
-      names = []
-      names << :GENERIC_READ if self.gr != 0
-      names << :GENERIC_WRITE if self.gw != 0
-      names << :GENERIC_EXECUTE if self.gx != 0
-      names << :GENERIC_ALL if self.ga != 0
-      names << :MAXIMUM_ALLOWED if self.ma != 0
-      names << :ACCESS_SYSTEM_SECURITY if self.as != 0
-      names << :SYNCHRONIZE if self.sy != 0
-      names << :WRITE_OWNER if self.wo != 0
-      names << :WRITE_DACL if self.wd != 0
-      names << :READ_CONTROL if self.rc != 0
-      names << :DELETE if self.de != 0
-      names
-    end
```

Comment:
This was added per #17212 about 2 and a half years ago but was never used. It now feels redundant with the new `#permissions` method that uses the abbreviated names that are aligned with the SDDL text.